x/payment/types/payfordata_test.go

```diff
@@ -151,7 +151,7 @@ func TestPowerOf2(t *testing.T) {
 	}
 }
 
-// TestCreateCommit only shows if something changed, it doesn't actually show
+// TestCreateCommitment only shows if something changed, it doesn't actually show
```

Comment:
[optional] while we're editing might as well fix the typo. might need to wrap this line
```suggestion
// TestCreateCommitment only shows if something changed, it doesn't actually show that
```